earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -285,90 +284,80 @@ void PacketGenAssetVersionImplD::UpdateChildren(
       prevTime = curTime;
 
       // populate inputs and inputVers & config.insets
+      packetLevelConfig.insets.reserve(mergeIndexes.size() + 1);
+      packetLevelConfig.insets.resize(mergeIndexes.size());
+      int threadsToUse = MiscConfig::Instance().PackGenThreads;
+      notify(NFY_WARN, "mergeIndexes: %lu", mergeIndexes.size());
       levInputs.reserve(mergeIndexes.size() * 2 + 1);
       levInputVers.reserve(mergeIndexes.size() * 2 + 1);
-      packetLevelConfig.insets.reserve(mergeIndexes.size() + 1);
-      for (std::vector<uint>::const_iterator idx = mergeIndexes.begin();
-           idx != mergeIndexes.end(); ++idx) {
-        const InsetInfo<ProductAssetVersion> &inset(*extra.insetInfo[*idx]);
-        const PacketGenInfo &genInfo(*extra.genInfo[*idx]);
-        AssetVersion blendVer;
+      levInputs.resize(mergeIndexes.size() * 2);
+      levInputVers.resize(mergeIndexes.size() * 2);
+      int i = 0;
+      std::vector<int> emptyElements;
+      emptyElements.reserve(mergeIndexes.size());
+      #pragma omp parallel for schedule(static) num_threads(threadsToUse)
```

Comment:
Can you add a comment explaining this pragma

Reply:
Will do.